AssetOptionsBinder.cs

```diff
@@ -96,6 +96,11 @@ internal class AssetOptionsBinder : BinderBase<AssetOptions>
             () => true,
             description: @"Skip assets that have been migrated already.");
 
+        private readonly Option<bool> _cleanInputAssets = new(
+           aliases: new[] { "--clean" },
+           () => true,
```

Comment:
can you double check with @weibz to see if the default is 'true'? (i'm guessing this means clean up by default?)